tools/clang/unittests/HLSLTestLib/D3DReflectionDumper.cpp

```diff
@@ -411,7 +502,7 @@ void D3DReflectionDumper::Dump(D3D12_SHADER_BUFFER_DESC &Desc) {
 }
 void D3DReflectionDumper::Dump(D3D12_SHADER_INPUT_BIND_DESC &resDesc) {
   SetLastName(resDesc.Name);
-  WriteLn("D3D12_SHADER_BUFFER_DESC: Name: ", m_LastName);
+  WriteLn("D3D12_SHADER_INPUT_BIND_DESC: Name: ", m_LastName);
```

Comment:
This and the other output change account for most of the changed files, but I can't really contrive an excuse not to make this change